data/mods/Magiclysm/npc/TALK_FORGE_HELEN_TAVREL.json

```diff
@@ -0,0 +1,112 @@
+[
+  {
+    "type": "npc",
+    "id": "forge_lord_reagents",
+    "//": "The Lord of the Forge of Wonders.",
+    "name_unique": "Helen Tavrel, Former Pirate",
+    "name_suffix": "Forge Lord",
+    "class": "NC_FORGE_LORD_REAGENT",
+    "attitude": 0,
+    "mission": 3,
+    "gender": "female",
+    "chat": "TALK_FORGE_LORD_REAGENT",
+    "faction": "forge_lords"
+  },
+  {
+    "type": "npc_class",
+    "id": "NC_FORGE_LORD_REAGENT",
+    "name": { "str": "Pirate Queen" },
+    "job_description": "My adventures would curdle your blood.",
+    "traits": [ { "group": "NPC_starting_traits" }, { "group": "Appearance_demographics" } ],
+    "//": "This is a unique NPC who doesn't get randomly selected background traits",
+    "common": false,
+    "bonus_int": { "one_in": 4 },
+    "bonus_per": { "one_in": 4 },
```

Comment:
How exactly does this work, BTW?

Reply:
I think it gives a 25% chance for an extra int or perception point.